core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/JsonSerializer.java

```diff
@@ -51,6 +60,8 @@ JsonObject writeTextAnnotation(TextAnnotation ta) {
         writeString("id", ta.getId(), json);
         writeString("text", ta.getText(), json);
         writeStringArray("tokens", ta.getTokens(), json);
+        if ( doWriteTokenOffsets )
+            writeTokenOffsets( TOKENOFFSETS, ta.getView(ViewNames.TOKENS), json);
```

Comment:
Similar behavior should be added to `readTextAnnotation`: 
https://github.com/mssammon/illinois-cogcomp-nlp-1/blob/03e49ed5aa539445579462c16410a1e2fd12ac7f/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/utilities/JsonSerializer.java#L116